### ISSUE TITLE
Man: Ch. 15. Library Messages — Pending Tasks

### ISSUE THREAD

**Issue by tajmone** [AnssiR66/AlanStdLib#105]
Chapter [15. Library Messages] is the old Ch. 10. on `my_game` instance, which was already discussed in #88 as being too long and needing to be reduced in size and require dynamic inclusion of StdLib source snippets to ensure that code excerpts will always mirror the current library status.

Also, now Ch. 15 is dedicated exclusively to the library messages (which were moved to the new module `lib_messages_library.i`) so we need to move somewhere else anything that doesn't strictly related to topic of verb responses and library messages.

- [ ] Chapter [15. Library Messages]:
    + [ ] The initial text "`My_game` is a so called meta-instance…" should be moved elsewhere. Also, the emphasis here should be that these attributes are really defined on the `DEFINITION_BLOCK` class, and inherited by the `my_game` instance, which can be used by end authors to override the default attributes strings.
    + [ ] All text referring to the Banner should be moved to [4. The Game Banner] — integrating it with what's already there, or discard it if redundant.
    + [ ] Sec. [§15.2. 1. Attributes for the start section] should be moved somewhere in the [Getting Started] Part.
    + [ ] Any text that doesn't pertain to library messages should be moved elsewhere, where it belongs appropriately.
    + [ ] The remaining text should be trimmed down drastically:
        * [ ] Instead of listing every possible message attribute defined by the library, this chapter should:
            - [ ] Cover the basic attributes that are really important to know — Sec. 15.3.2/3/4 are worth keeping, for they are useful to every game.
            - [ ] Provide a detailed tutorial on how to track down a specific message and override it via the `my_game` instance. The reader should be advised to consult the `lib_messages_library.i` source module to learn all the possible messages.
                + [ ] Any specific excerpts needed from `lib_messages_library.i`  should be imported directly from the source file, by adding the required AsciiDoc region tags for selective inclusion.
        * [ ] We could improve the comments in `lib_messages_library.i` source to make it more user friendly (this was already parly done in the meantime), maybe pasting therein some of the original text from the Manual that we're about to remove, and we might also better organize the message-attributes by categories, providing them with headers to make it easier to find attributes.

[15. Library Messages]: http://htmlpreview.github.io/?https://github.com/AnssiR66/AlanStdLib/blob/dev-2.2.0/extras/manual/StdLibMan.html#ch.library-messages
[§15.2. 1. Attributes for the start section]: http://htmlpreview.github.io/?https://github.com/AnssiR66/AlanStdLib/blob/dev-2.2.0/extras/manual/StdLibMan.html#_1_attributes_for_the_start_section

[Getting Started]: http://htmlpreview.github.io/?https://github.com/AnssiR66/AlanStdLib/blob/dev-2.2.0/extras/manual/StdLibMan.html#part.getting-started
[4. The Game Banner]: http://htmlpreview.github.io/?https://github.com/AnssiR66/AlanStdLib/blob/dev-2.2.0/extras/manual/StdLibMan.html#ch.game-banner

**Comment by tajmone** [comment]
@AnssiR66, if you could help with all the content edits mentioned above, I'd really be grateful. I haven't find the time to tackle with this chapter yet, and am not very confident with it either.

I can handle all the AsciiDoc region tags and selective inclusions, so don't worry about these nitty gritty details, just focus on reducing the contents size, adding the tutorial, and advising me on where the removed contents should be move to (if not discarded), and which parts of the original text should be moved to the documenting comments of `lib_messages_library.i`, before removal from the Manual.

**Comment by AnssiR66** [comment]
Yes, sure! I will take a look at the passages you suggested, and provide something in the very near future.
